.gitattributes

```diff
@@ -1,8 +1,12 @@
-/.github/ export-ignore
-/bin/ export-ignore
-/tests export-ignore
-/.gitattributes export-ignore
-/.gitignore export-ignore
-/package.xml.tpl export-ignore
-/phpdoc.ini.dist export-ignore
-/phpunit.xml.dist export-ignore
+/.github/           export-ignore
+/bin/               export-ignore
+/tests              export-ignore
+/.editorconfig      export-ignore
+/.gitattributes     export-ignore
+/.gitignore         export-ignore
+/CHANGELOG.md       export-ignore
```

Comment:
I'm wondering if it actually might make sense to allow CHANGELOG.md in the export?

Reply:
Yeah, I think I would keep the CHANGELOG and the README in the distribution package